src/app/static/src/tests/integration/modelOptions.itest.ts

```diff
@@ -40,19 +39,24 @@ describe("model options actions integration", () => {
         expect(commit.mock.calls[2][0]["type"]).toBe("SetModelOptionsVersion");
         expect(commit.mock.calls[2][0]["payload"]).toBeDefined();
     });
-/*
+
+    
     it("can validate model options", async () => {
         const commit = jest.fn();
-        const options = jest.fn()
-        const version = jest.fn();
-    
-        await mockAxios.onPost("/model/validate/options",{options, version}).reply(200, "Success")
-        //await actions.validateModelOptions({commit, mockState, rootState} as any, {options,version} as any);
-
-        expect(commit.mock.calls[1][0]["type"]).toBe(ModelOptionsMutation.Validate);
-        //const payload = commit.mock.calls[1][0]["payload"];
-        //expect(payload).toBe(false);
+        const options =  jest.fn();
+        const version = jest.fn()
+      
+        const mockState = {
+            language: Language.en,
+            modelOptions: {
+                options: {},
+                version: {}
+            }
+        }
+        //passed mock params which will return validation error
+        await actions.validateModelOptions({commit, rootState: mockState} as any, {options,version} as any);
+        expect(commit.mock.calls[0][0]).toBe(ModelOptionsMutation.LoadUpdatedOptions);
+        expect(commit.mock.calls[1][0]["type"]).toBe(ModelOptionsMutation.HasValidationError);
+        expect(commit.mock.calls[0][1]).toBeDefined();
```

Comment:
I'm sorry this is still a bit vague - it could have error-ed because e.g. the route wasn't found. I think checking the error detail will be able to tell us whether the correct endpoint was hit.